bundles/org.openhab.binding.mybmw/src/main/resources/OH-INF/thing/range-channel-types.xml

```diff
@@ -26,26 +26,36 @@
 	<channel-type id="soc-channel">
 		<item-type>Number:Dimensionless</item-type>
 		<label>Battery Charge Level</label>
-		<state pattern="%d %unit%" readOnly="true"/>
+		<state pattern="%d %%" readOnly="true"/>
 	</channel-type>
 	<channel-type id="remaining-fuel-channel">
 		<item-type>Number:Volume</item-type>
 		<label>Remaining Fuel</label>
 		<state pattern="%d %unit%" readOnly="true"/>
 	</channel-type>
+	<channel-type id="estimated-fuel-l-100km-channel">
+		<item-type>Number:Dimensionless</item-type>
```

Comment:
Why Dimensionless, shouldn't this rather be a plain Number?

Reply:
hmm I understood that in OH4 all numbers should have dimensions due to UoM. But I can remove it. I see right now no disadvantage in defining it as dimensionless and maybe I misunderstood this part.

Reply:
`Number:Dimensionless` has a default abstract unit one, so I think for this case a plain `Number` would be more appropriate. Dimensionless is used specifically for %, ppm and dB - see:
https://www.openhab.org/docs/concepts/units-of-measurement.html#list-of-units

It would be quite unlikely that anyone would want to enforce unit %, ppm or dB for this channel, so you don't really gain anything from having it Dimensionless.